code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -255,15 +255,17 @@ GLOBAL_LIST_INIT(cardboard_recipes, list ( \
 /obj/item/stack/sheet/cardboard/attackby(obj/item/I, mob/user, params)
 	if(!istype(I, /obj/item/stamp/clown))
 		return ..()
-	if(get_amount() >= 2)
-		to_chat(user, "<span class ='notice'>You only need one cardboard sheet for this.</span>")
-		return
-	if(!user.temporarilyRemoveItemFromInventory(src))
-		return
-	to_chat(user, "<span class='notice'>You stamp the cardboard! Its a clown box! Honk!</span>")
+	if(istype(loc, /obj/item/storage))
+		to_chat(user,"<span class='notice'>Take the [src] out first!</span>")
+		return ..()
+	var/turf/T = get_turf(src.loc)
 	playsound(src, 'sound/items/bikehorn.ogg', 50, 1, -1)
-	new/obj/item/storage/box/clown(drop_location())
-	use(1)
+	to_chat(user, "<span class='notice'>You stamp the cardboard! Its a clown box! Honk!</span>")
+	new/obj/item/storage/box/clown(drop_location(T))
+	if ((src.amount > 1) && (user.is_holding(src))) //prevent a bug when holding stack
+		amount-- //you also keep the stack in your hand
+	else
+		use(1)
```

Comment:
Just replace this whole proc with
```
if(istype(I, /obj/item/stamp/clown) && use (1))
    to_chat(user, "<span class='notice'>You stamp the cardboard! Its a clown box! Honk!</span>")
    new/obj/item/storage/box/clown(drop_location())
else
    . = ..()
```

Reply:
Aight. Just gotta fix a couple bugs first, and i'll commit it.

Reply:
use(1) is still returning too soon - it won't proc anything with a single sheet.
We need to move the sound and text message above the if(use).
It also prevents the drop_location() as it can't read where it is, since the source gets deleted.

Unless you can find a way to memorize the location on another object, we're gonna have to use the turf as a placeholder. Its not optimal, but it'll work.

Reply:
Aright, you're right, I forgot QDEL  immediately changes it's loc, give me a few minutes I'll fix it

Reply:
I was about to push this: 
<script src="https://gist.github.com/Mercenaryblue/db910be0f6285e068819277114bacaab.js"></script>

Reply:
```
if(istype(I, /obj/item/stamp/clown))
    var/atom/droploc = drop_location()
    if(use(1))
        playsound(I, 'sound/items/bikehorn.ogg', 50, 1, -1)
        to_chat(user, "<span class='notice'>You stamp the cardboard! Its a clown box! Honk!</span>")
        new/obj/item/storage/box/clown(drop_location())
else
    . = ..()
```
    
This should work.  I've changed it to make the horn honk rather than the box, (Edge case involving TK mutation)

Reply:
Onward to testing!